### DISCUSSION TITLE
How to change token when sending a transaction (right now it works just for ETH) on WalletConnectModal

### DISCUSSION THREAD

**Discussion by fandreirebaseit** [WalletConnect/walletconnect-monorepo#3485] [Developer Support]
Hi,

I have migrated to wallet connect modal v2 on react native but the transactions are only working on ETH token, the SESSION_PARAMS indicate just methods which involves eth_sign, eth_transactions etc.
How could I modify or send another token to the wallet when calling sendTransaction method?

**Comment by mshahjahan-mufin** [comment]
@fandreirebaseit If you get the solution let me know as I am stuck with same problem

**Comment by glitch-txs** [comment]
hi, which pacakge are you using?

**Reply by mshahjahan-mufin** [reply]
WalletConnecV2 for Flutter and WalletConnectModal for Flutter
